mk_verificator/tests/test_migration.py

```diff
@@ -44,3 +42,4 @@ def test_migration_for_all_nodes(nova_client, create_vm):
     for host in list_hosts:
         create_vm.live_migrate(host=host)
         checking_vm_host(nova_client, create_vm.id, host)
+    create_vm.delete()
```

Comment:
why did you delete yield??? that's more useful than keep object deletion in "unsafe" code block